web/submission/views.py

```diff
@@ -57,17 +57,22 @@
 logger = logging.getLogger(__name__)
 
 
-def get_form_data(platform):
```

Comment:
removed platform arg, as we were already listing all machines anyway

Reply:
thanks for clarification